File: data/code_pairs/cpp/1317.cpp
// file: Handlers.hpp
#include <iostream>

class Handlers {
public:
    static void onClosing() {
        std::cout << "Closing event triggered." << std::endl;
    }
};

// file: main.cpp
#include "Handlers.hpp"

int main() {
    Handlers::onClosing();
    return 0;
}

File: data/code_pairs/fortran/1317.f90
! file: handlers.f90
module handlers
  implicit none
contains
  subroutine onClosing()
    print *, "Closing event triggered."
  end subroutine onClosing
end module handlers

! file: main.f90
program main
  use handlers
  implicit none
  call onClosing()
end program main